apps/emqx_ldap/src/emqx_ldap.erl

```diff
@@ -76,7 +76,20 @@ fields(config) ->
                 desc => ?DESC(request_timeout),
                 default => <<"5s">>
             })}
-    ] ++ emqx_connector_schema_lib:ssl_fields().
+    ] ++ emqx_connector_schema_lib:ssl_fields();
+fields(bind_opts) ->
+    [
+        {bind_password,
+            ?HOCON(
+                binary(),
+                #{
+                    desc => ?DESC(bind_password),
+                    default => <<"${password}">>,
+                    example => <<"${password}">>,
+                    validator => fun emqx_schema:non_empty_string/1
+                }
```

Comment:
Q: I see this can be a placeholder, but would it be common for an user to set it to a concrete value?  If so, we might want to mark this as `sensitive`.